### PR TITLE
Costy branch

### PR DESCRIPTION
**Pull request by costy619**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -20,14 +19,15 @@
...
         <activity android:name=".MainActivity"/>
+        <activity android:name=".LogInPerson">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-        <activity android:name=".MainActivity" />
```
Comment (Elenaadochitei) [inline app/src/main/AndroidManifest.xml:29]: why did you delete this line?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -20,14 +19,15 @@
         <activity android:name=".CancelPerson" />
```
Comment (Elenaadochitei) [inline app/src/main/AndroidManifest.xml:19]: Missing the line:
<activity android:name=".MyAccount"></activity>

**Review by Ana0809 — APPROVED** [review]
